great-idea-website/css/index.css

```diff
@@ -63,4 +63,106 @@ h1, h2, h3, h4, h5 {
     margin-bottom: 15px;
 }
 
-/* Your code starts here! */
\ No newline at end of file
+/* Your code starts here! */
+nav a{
+	color: grey;
+	padding: 20px;
+	text-decoration: none;
+	display: inline-block;
+}
+
+nav .logo{
+	vertical-align: middle;
+	display: inline-block;
+}
+
+.header div h1{
+	font-size: 60px;
+	text-align: center;
+	margin: 80px 30px 0;
+	padding: 0;
+	display: inline-block;
+	position: relative;
+}
+
+.header img{
+	padding: 20px;
+	margin: 0 10px 0 300px;
+	border-radius: 10px;
+}
+
+.header div{
+	position: absolute;
+	display: inline-block;
+}
+
+button{
+	display: block;
+	padding: 5px;
+	margin: 0 0 0 95px;
+	height: 30px;
+	width: 120px;
+	box-sizing: border-box;
+	background-color: white;
+	border-style: ridge;
+	text-align: center;
+	position: relative;
+}
+
+
+h2{
+	margin-top: 20px;
+	padding: 10px 0;
+}
+
+section{
+	margin: 20px 0;
+	padding: 10px;
+}
+
+.features{
+	border-top: 1px solid black;
+	margin: 20px 30px;
+}
+.bottom{
+	border-bottom: 1px solid black;
+	margin: 20px 30px;
+}
+
+section div{
+	padding: 10px;
+	max-width: 30%;
+	margin: 5px;
+}
+
+
+.features div{	
+	padding: 10px;
+	max-width: 45%;
+	margin: 0 auto;
+	display: inline-block;
+	text-align: left;
+}
+
+.bottom div{
+	display: inline-block;
+	text-align: left;
+}
+
+.contact{
+	margin: 0 30px;
+}
+
+.middle-img{
+	display: block;
+	margin: 20px 30px;
+}
+
+footer{
+	text-align: center;
+}
+
+*{
+	max-width: 80%;
+	margin: 0 auto;
+}
```

Comment:
It looks like we are missing that `.container` element that we worked on yesterday that helped us box up all of our content in a neat package, and center it all on screen. So at the moment everything is lined up on the left hand side, as opposed to being centered. 

Let's wrap up all of those elements in a `<div class="container">` and then add those corresponding styles that we talked about in our CSS file, and that should fix everything up!